rio_tiler/io/base.py

```diff
@@ -27,35 +29,35 @@ class SpatialMixin:
 
     Attributes:
         tms (morecantile.TileMatrixSet, optional): TileMatrixSet grid definition. Defaults to `WebMercatorQuad`.
-        bbox (tuple): Dataset bounds (left, bottom, right, top). **READ ONLY attribute**.
-        minzoom (int): Overwrite Min Zoom level. **READ ONLY attribute**.
-        maxzoom (int): Overwrite Max Zoom level. **READ ONLY attribute**.
+        minzoom (int): Dataset Min Zoom level. **READ ONLY attribute**.
+        maxzoom (int): Dataset Max Zoom level. **READ ONLY attribute**.
+        bounds (tuple): Dataset bounds (left, bottom, right, top). **READ ONLY attribute**.
+        crs (rasterio.crs.CRS): Dataset crs. **READ ONLY attribute**.
 
     """
 
     tms: TileMatrixSet = attr.ib(default=WEB_MERCATOR_TMS)
+    minzoom: int = attr.ib(default=None)
+    maxzoom: int = attr.ib(default=None)
```

Comment:
we put Min/Max zoom in `init` for visibility (**breaking**)

Reply:
Nope, I removed this
1. not related to the goal of the PR
2. unnecessary breaking change